src/legacy/ui/public/inspector/ui/inspector_view.tsx

```diff
@@ -17,29 +17,20 @@
  * under the License.
  */
 
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
-
 import { EuiFlyoutBody } from '@elastic/eui';
 
 /**
- * The InspectorView component should be the top most element in every implemented
- * inspector view. It makes sure, that the appropriate stylings are applied to the
- * view.
+ * Do not use this, use {@link EuiFlyoutBody} EUI component directly.
+ *
+ * @deprecated
  */
-const InspectorView: React.SFC<{ useFlex?: boolean }> = ({ useFlex, children }) => {
-  const classes = classNames({
-    'kbnInspectorView--flex': Boolean(useFlex),
-  });
-  return <EuiFlyoutBody className={classes}>{children}</EuiFlyoutBody>;
-};
+export const InspectorView: React.FC<{ useFlex?: boolean }> = ({ useFlex, children }) => {
+  const style: React.CSSProperties = {};
 
-InspectorView.propTypes = {
-  /**
-   * Set to true if the element should have display: flex set.
-   */
-  useFlex: PropTypes.bool,
-};
+  if (useFlex) {
+    style.display = 'flex';
```

Comment:
The problem with switching away from SASS here, is that you're not using the prefixer to automatically add the fallbacks for older/other browsers. I would highly suggest staying with SASS for now until a better CSS in JS solution has been found.

Reply:
Does `display: flex` need prefixing?

Reply:
Yes, mainly for IE, but our prefixer also adds the `-webkit-box` prefix for any older webkit browsers. https://caniuse.com/#feat=flexbox

Reply:
> Yes, mainly for IE

I don't think we need Flexbox prefixing for IE, because IE11 supports the standard `display: flex`, the `-ms-flex` value is needed for IE10, which we don't support, I believe. ([Also with recent CSP changes, it looks like we don't really support IE11, at least not the same as other browsers](https://github.com/elastic/kibana-team/issues/20).)

I believe we don't support IE10, if that is indeed the case, I suggest we should fix our `.browserlist` rules instead, so that we don't ship unnecessarily large CSS files with prefixes we don't need.

> ... but our prefixer also adds the -webkit-box prefix for any older webkit browsers.

Do we need that though? `display: box` is not the same as `display: flex`, maybe it is not good idea to use `box` at all.

Do you know for which Webkit browsers we need that for? For example, Chrome supports the standard `display: flex` since Chrome 21, and Chrome 21 and below have virtually zero market share.

Reply:
We still support IE11 till the release of 8.0. CSP change is only in master (which is 8.x).

Beyond prefixing, we should continue writing Sass files and not creating style tags except for places that need them (ex: things that need custom calculated values). We do this mostly for maintenance reasons (it's easier to cascade through imported sass files than change individual style tags in JS). We've seen when style tags are used, even minimally like this, people just take the opportunity to write on top of them and follow the pattern they see in the file they are looking at. Also, I know this sounds like a terrible reason, but we actually get alerts on Sass changes, and we don't on style tags in JS. There's dozens of PRs changing styling every day and it's a nice smoke test to have some knowledge of what's changing day to day. The next change to this file we might not know about.

All this may (hopefully!) change in the future as we implement more robust styling systems, but at least for now it would really help to continue with the process / systems we currently have in place.

Reply:
++ To @snide's comments about why we prefer SASS over JS in CSS. 

As to prefixing, EUI (and subsequently Kibana) is currently setup to use `autoprefixer` at `'last 2 versions'` which is a good basic default. It could be that the library isn't as up to date as we'd like. However, changing this setting now could possibly mean some major visual regressions to some customers on older browsers. It's a very quick win to not alienate those who can't currently upgrade their browsers.

Reply:
@cchaos @snide I actually removed this `<InspectorView>` component entirely, as it is not needed anymore and thus this CSS is now removed, too. Could you please give this PR another look, as Design Team is marked as code owners in this PR?